src/setupTests.js

```diff
@@ -9,4 +9,4 @@ import '@testing-library/jest-dom/extend-expect';
 import MutationObserver from '@sheerun/mutationobserver-shim';
 window.MutationObserver = MutationObserver;
 
-import './testServer';
+// import './testServer';
```

Comment:
@cfernando415 i'm no familiar with MSW, do you know if there's an option to whitelist certain tests? it appears the chromium browser won't run with the mock server